akka-docs/src/main/paradox/scala/distributed-pub-sub.md

```diff
@@ -30,9 +30,13 @@ any other node.
 There a two different modes of message delivery, explained in the sections
 [Publish](#distributed-pub-sub-publish) and [Send](#distributed-pub-sub-send) below.
 
+@@@ div { .group-scala }
+
 A more comprehensive sample is available in the
 tutorial named [Akka Clustered PubSub with Scala!](https://github.com/typesafehub/activator-akka-clustering).
 
```

Comment:
No equivalent Java example exists.

As activator is deprecated, should we rather remove the reference to https://github.com/typesafehub/activator-akka-clustering?